llvm/test/CodeGen/X86/tls-function-argument.ll

```diff
@@ -0,0 +1,30 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=x86_64 -verify-machineinstrs < %s -relocation-model=pic | FileCheck %s
```

Comment:
< %s to th eend, don't need -verify-machineinstrs

Reply:
I think -verify-machineinstrs is useful here: Without my patch, the test fails in the MachineVerifier, where the call stack pseudos are checked. Without -verify-machineinstrs, this would only happen in builds with expensive checks enabled, and the test would be ineffective for other builds.